src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+type Props = {
+  tabs: Tab[],
+  selectedTab: Tab,
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTab,
+  onTabSelected,
+}) => (
+  <div>
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li key={tab.id}>
+            <a
+              href={`#${tab.id}`}
+              className={classNames({ 'is-active': tab.id === selectedTab.id })}
```

Comment:
doesn't seem like you have this class
![image](https://user-images.githubusercontent.com/37161315/180228139-8ff05993-7ccc-411e-99c7-1ef92e360045.png)